assets/js/modules/search-console/components/dashboard/DashboardImpressionsWidget.js

```diff
@@ -88,14 +89,33 @@ function DashboardImpressionsWidget() {
 		return <ReportZero moduleSlug="search-console" />;
 	}
 
-	const { totalImpressions, totalImpressionsChange, dataMap } = extractSearchConsoleDashboardData( data );
+	// Split the data in two chunks.
+	const half = Math.floor( data.length / 2 );
+	const latestData = data.slice( half );
+	const olderData = data.slice( 0, half );
+
+	const totalImpressions = sumObjectListValue( latestData, 'impressions' );
+	const totalOlderImpressions = sumObjectListValue( olderData, 'impressions' );
+	const totalImpressionsChange = changeToPercent( totalOlderImpressions, totalImpressions );
+
+	const sparklineData = [
+		[
+			{ type: 'string', label: 'Day' },
+			{ type: 'number', label: 'Clicks' },
+		],
+		...extractForSparkline( latestData, 'impressions', 'keys.0' ).map( ( row ) => {
+			const date = new Date( row[ 0 ] );
+			// Sparkline data needs headers and dates formatted as MM/DD
+			return [ date.getMonth() + 1 + '/' + date.getUTCDate(), row[ 1 ] ];
```

Comment:
Same here re: string interpolation. Let me know if you disagree, that's fine 😄 
```suggestion
			return [ `${ date.getMonth() + 1 }/${ date.getUTCDate() }`, row[ 1 ] ];
```